Examples/ArpSpoofing/CMakeLists.txt

```diff
@@ -0,0 +1,11 @@
+add_executable(ArpSpoofing main.cpp)
```

Comment:
I think we should remove the Makefile from all the examples

Reply:
removed